tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/wait_instruction.h

```diff
@@ -0,0 +1,136 @@
+/**
+ * @file wait_instruction.h
+ * @brief
+ *
+ * @author Levi Armstrong
+ * @date November 15, 2020
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2020, Southwest Research Institute
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef TESSERACT_COMMAND_LANGUAGE_WAIT_INSTRUCTION_H
+#define TESSERACT_COMMAND_LANGUAGE_WAIT_INSTRUCTION_H
+
+#include <tesseract_common/macros.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <iostream>
+#include <string>
+#include <tinyxml2.h>
+TESSERACT_COMMON_IGNORE_WARNINGS_POP
+
+#include <tesseract_command_language/instruction_type.h>
+
+namespace tesseract_planning
+{
+enum class WaitInstructionType : int
+{
+  TIME = 0,
+  DIGITAL_INPUT_HIGH = 1,
+  DIGITAL_INPUT_LOW = 2,
+  DIGITAL_OUTPUT_HIGH = 3,
+  DIGITAL_OUTPUT_LOW = 4
+};
+
+class WaitInstruction
```

Comment:
I'll let Colin review this, but I'd request documentation on what this is and how to use it.

Reply:
I added documentation.